src/braket/experimental/autoqasm/pulse/pulse.py

```diff
@@ -115,13 +114,35 @@ def play(frame: Frame, waveform: Waveform) -> None:
     _pulse_instruction("play", frame, waveform)
 
 
-def capture_v0(frame: Frame) -> None:
+def capture_v0(frame: Frame) -> BitVar:
     """Adds an instruction to capture the bit output from measuring the specified frame.
 
     Args:
         frame (Frame): Frame on which the capture operation needs to be performed.
+    Returns:
+        BitVar: A BitVar with the result of the capture.
     """
-    _pulse_instruction("_capture_v0_with_return", frame)
+
+    def _add_sequence() -> BitVar:
+        _validate_uniqueness(pulse_sequence._frames, frame)
+        extern_call = oqpy.declare_extern("capture_v0", [("frame", oqpy.FrameVar)], oqpy.bit)
+        bit_var = BitVar()
+        pulse_sequence._program.set(bit_var, extern_call(frame))
+        pulse_sequence._capture_v0_count += 1
+        pulse_sequence._frames[frame.id] = frame
+        return bit_var
+
+    aq_context = aq_program.get_program_conversion_context()
+    aq_context._has_pulse_control = True
+
+    pulse_sequence = PulseSequence()
+    pulse_sequence._program = aq_context.get_oqpy_program(mode=aq_program.ProgramMode.PULSE)
+
+    if aq_context._calibration_definitions_processing:
```

Comment:
@yitchen-tim do we / should we support measurement calibrations?

Reply:
Non of Braket QPU currently supports it, but I think AutoQASM should support it given that openqasm allows it.

Reply:
ty -- I will add a test for it and that should fix coverage (not that measurement level 0 as a measurement calibration is meaningful, but 🤷‍♀️)

Reply:
We don't currently support this (attempting, without return statement, yields ParameterTypeError) so I will open an issue instead and remove this code

Reply:
https://github.com/orgs/amazon-braket/projects/2/views/1?pane=issue&itemId=54800501